tendermint/src/lite/verifier.rs

```diff
@@ -377,15 +377,16 @@ mod tests {
     // create an initial trusted state from the given vals
     fn init_trusted_state(
         vals_vec: Vec<usize>,
+        commit_vec: Vec<usize>,
```

Comment:
Hmm, now that we use a pair `ValsAndCommit` below, should we use it here too?

Reply:
Yes, will do that 👍